### PR TITLE
[SGM-6237] : Updates to Home and Cody Page

### PR DESCRIPTION
**Pull request by gitstart-sourcegraph**
## Description
Update Home and Cody Page

## Ref
[SG Issue](https://github.com/sourcegraph/about/issues/6237)
[Gitstart ticket](https://clients.gitstart.com/sourcegraph/1/tickets/SGM-6237)

## Demo

https://user-images.githubusercontent.com/89894075/236512264-6ec678ef-fe9e-4eca-9e31-47b38c4f6daf.mov

### CODE REVIEW

**Comment by fabicastp** [comment]
- [ ] Looks like the main navigation CTA still needs to be updated to “Get Cody for free” and direct to the sign-up page
- [ ] GIF container in the hero should have the 8px linear gradient stroke and 8px rounded corners
- [ ] Background illustration needs to be updated so it doesn’t overlap with the copy (reflected in the mockup)
- [ ] Space between app section and the 3 feature blocks above should only be 24px

**Comment by fabicastp** [comment]
- [ ] Update the form thank you message to: Thank you! We'll get back to you soon. 

Currently, it gives the option to download the big code report. 

Screenshot 2023-05-05 at 2 20 52 PM

**Comment by traceyljohnson** [comment]
- gradient border on the GIF is not rendering well. If it helps, we can simplify to just a solid `Violet-500` at 50% opacity. 

Once fixed, should be good to ship!

File: src/styles/globals.css [hunk 1]
```diff
@@ -279,6 +279,15 @@
     .sg-bg-gradient-forrester {
         background: radial-gradient(90.85% 205.59% at -43.06% 143.19%, rgba(171, 179, 193, 0.63) 0%, rgba(171, 179, 193, 0) 100%);
     }
+
+    .sg-video-border-gradient {
+        border-color: hsla(273, 80%, 35%, 0.5);
+        background: linear-gradient(90deg, rgba(98, 18, 163, 0.5) 0.41%, rgba(39, 7, 65, 0.5) 54.04%, rgba(30, 6, 50, 0.5) 94.78%);
```
Comment (gitstart-sourcegraph) [inline src/styles/globals.css:285]:
```suggestion
```

**Review by fabicastp — APPROVED** [review]

**Review by iskyOS — APPROVED** [review]

**Pull request merged**